packages/components/src/presentation-components/tree/controlled/PresentationTreeRenderer.tsx

```diff
@@ -70,6 +72,7 @@ export function useFilterablePresentationTree({ nodeLoader }: useFilterablePrese
         onApply={(info) => {
           info === undefined ? clearFilter(filterNode.id) : applyFilter(filterNode.id, info);
           setFilterNode(undefined);
+          onFilterApplied?.();
```

Comment:
I don't think callback name matches when it's called. Because it will be also called when filter is removed. If we are only interested in filter being applied I think we should not invoke `onFilterApplied` when `info` is `undefined`.

Reply:
My intention was to have it be called on filter clear too but it seems that is not required so changed to only call when a filter is added.